src/tools/run-make-support/src/cc.rs

```diff
@@ -2,7 +2,9 @@ use std::env;
 use std::path::Path;
 use std::process::Command;
 
-use crate::{bin_name, cygpath_windows, handle_failed_output, is_msvc, is_windows, tmp_dir, uname};
+use crate::{
+    bin_name, cygpath_windows, handle_failed_output, is_msvc, is_windows, tmp_path, uname,
```

Comment:
Suggestion: we should really rename `tmp_dir` and `tmp_path` to something that describes the semantics better. I don't have a good name yet, I'll need to review the semantics of the `$TMPDIR` as is implemented in compiletest and used by run-make tests.

Reply:
Suggestion: in https://github.com/rust-lang/rust/pull/125827, we called the output artifacts directory `rmake_out`. Maybe we should rename `tmp_dir` -> `rmake_out_dir` and `tmp_path` to `rmake_out_path`?